v2/features/farms/FarmsTableRowHeader.tsx

```diff
@@ -96,7 +161,11 @@ const FarmsTableRowHeader: FC<Props> = ({ jar, simple, open }) => {
         </div>
       </RowCell>
       <RowCell>
-        <p className="font-title font-medium text-base leading-5">{aprRangeString}</p>
+        <span className="font-title font-medium text-base leading-5">{aprRangeString}</span>{" "}
+        <MoreInfo secondaryText={ReactHtmlParser(apyRangeTooltipText)} />{" "}
```

Comment:
@larrythecucumber321 I think this would be cleaner if we refactor the `MoreInfo` component to accept a React element instead since the current API isn't flexible enough. Doing that will allow us to format the content of the tooltip however we want (e.g. I think adding bold and/or some color to the new APY tooltip content would make it look much better). The PR looks great, I can take it from here and merge it with that refactor.

Reply:
Thanks @janklimo - look forward to seeing you work your magic